src/js/Controllers/UIController.js

```diff
@@ -190,12 +190,14 @@ class UIController {
                 ))
                 this.onSystemContext("MENU", rpc.params.appID)
                 return true
-            case "OnButtonSubscription":
+            case "SubscribeButton":
                 store.dispatch(subscribeButton(
                     rpc.params.appID,
-                    rpc.params.name,
-                    rpc.params.isSubscribed
+                    rpc.params.buttonName,
+                    true
                 ))
+                const appID = store.getState().activeApp;
+                this.listener.send(RpcFactory.SubscribeButton(appID, rpc.params.buttonName));
```

Comment:
@LiliiaShlikhtLuxoft Looks like response will send for active app only. Is it true? If so, is it expected behavior? Seems response should be sended for requested app id, not for active only

Reply:
@ValeriiMalkov thank you! Fixed